.travis/clangformat.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
@amoss, could you please use `#!/usr/bin/env bash` here?

Thanks.

Reply:
Will do.